pandas/core/generic.py

```diff
@@ -1303,8 +1303,81 @@ def __invert__(self):
 
     def equals(self, other):
         """
-        Determines if two NDFrame objects contain the same elements. NaNs in
-        the same location are considered equal.
+        Test whether two NDFrame objects contain the same elements.
```

Comment:
I think you can leave `Test whether two objects contain the same elements.` for now.

The problem is that NDFrame is not a class we expect users to know about (it's private). So, it shouldn't be used in the documentation. `Series or DataFrame` or `object` are probably what we want to use in most cases.